src/serverless-openapi-typescript.ts

```diff
@@ -183,8 +184,23 @@ export default class ServerlessOpenapiTypeScript {
         const outputFile = this.serverless.processedInput.options.output;
         const openApi = yaml.load(fs.readFileSync(outputFile));
         this.patchOpenApiVersion(openApi);
-        this.tagMethods(openApi);
-        fs.writeFileSync(outputFile, outputFile.endsWith('json') ? JSON.stringify(openApi, null, 2) : yaml.dump(openApi));
+        this.enrichMethodsInfo(openApi);
+        const encodedOpenAPI = this.encodeOpenApiToStandard(openApi);
+        fs.writeFileSync(outputFile, outputFile.endsWith('json') ? JSON.stringify(encodedOpenAPI, null, 2) : yaml.dump(encodedOpenAPI));
+    }
+
+    encodeOpenApiToStandard(openApi) {
+        const INVALID_CHARACTERS_KEY = /<|>/g;
+        const INVALID_CHARACTERS_REF = /%3C|%3E/g;
+
+        const mapObject = mapKeysDeep(openApi, (value, key) =>
+            INVALID_CHARACTERS_KEY.test(key) ? key.replace(INVALID_CHARACTERS_KEY, '_') : key
+        );
+
+        return mapValuesDeep(mapObject, (value, key) =>
+            isString(value) && key === '$ref' && INVALID_CHARACTERS_REF.test(value) ?
```

Comment:
I think `isString(value)` is superfluous.  
If `key === '$ref'`, what else could it be?

Reply:
For safety - but we don't need it

Reply:
8036b4b